### DISCUSSION TITLE
[StylexPlugin] Get the name of the generated CSS file.

### DISCUSSION THREAD

**Discussion by deadalnix** [facebook/stylex#434] [Q&A] [answered]
### Describe the feature request

Hi,

When using StylexPlugin, the css is not automatically injected in the html of the page. The workaround is to simply add the CSS manually in the html template, and this works when the name of the CSS file is known.

However, it is common practice to add a hash of the content in the name of the CSS file, which ensure various caching layers are defeated when, the content changes, even when the CSS is cached aggressively.

In that scenario, it doesn't seems to be possible to add the CSS file in the template in any way, as far as I can tell, because its name is unknown. While fancy solutions are being worked on, it'd be tremendously useful to just have a way to get the generated file name out of the plugin, so we can stick it in the template.

**Comment by nmn** [comment, marked as answer]
Please see the RFC for a better Webpack plugin. There are techniques mentioned there which should help solve this problem.

**Reply by deadalnix** [reply]
I ended up creating a css bundle for the global style and appending to it. Not ideal, but it does the job.

**Reply by nmn** [reply]
This is the best that can be done until the plugin in rewritten.
